pymc/logprob/abstract.py

```diff
@@ -131,14 +143,21 @@ def _icdf_helper(rv, value, **kwargs):
     return rv_icdf
 
 
-class MeasurableVariable(abc.ABC):
-    """A variable that can be assigned a measure/log-probability"""
+class MeasurableOp(abc.ABC):
+    """An operation whose outputs can be assigned a measure/log-probability"""
+
 
+MeasurableOp.register(RandomVariable)
 
-MeasurableVariable.register(RandomVariable)
 
+class MeasurableOpMixin(MeasurableOp):
+    """MeasurableOp Mixin with a distinctive string representation"""
 
-class MeasurableElemwise(Elemwise):
+    def __str__(self):
+        return f"Measurable{super().__str__()}"
```

Comment:
This allows seeing when a Measurable replacement was actually performed in the IR graph. Most times we use a subclass of the pre-existing Op, and it would not be distinguishable in the string representation